src/constants.py

```diff
@@ -6,3 +6,5 @@
 FLASK_CONTAINER_NAME = "flask-app"
 FLASK_SERVICE_NAME = "flask-app"
 FLASK_ENV_CONFIG_PREFIX = "FLASK_"
+FLASK_DATABASE_NAME = "flask-app"
+FLASK_SUPPORTED_DATABASES = ("mysql", "postgresql")
```

Comment:
Wouldn't it be better to read that dynamically from the metadata ?
Don't think it'll require a lot of work while preparing for the next steps

Reply:
Done